elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreHelper.java

```diff
@@ -261,4 +268,44 @@ private void validateHavingClause(FilterExpression havingClause) {
             validateHavingClause(((NotFilterExpression) havingClause).getNegated());
         }
     }
+
+    /**
+     * Method to verify that all the sorting options provided
+     * by the user are valid and supported.
+     */
+    public void validateSorting() {
+        Sorting sorting = entityProjection.getSorting();
+        if (sorting == null) {
+            return;
+        }
+        Set<String> allFields = getRelationships();
+        allFields.addAll(getAttributes());
```

Comment:
This is the second time we have this logic (check line 166), I think we can make it into a method like `getAllFields()`.